test/cypress/integration/settings/payment-types.cy.js

```diff
@@ -53,6 +53,104 @@ describe('Payment types', () => {
       .should('contain', 'Accepted card types have been updated')
   })
 
+  context('when the account is test', () => {
+    context('when 3DS is not enabled', () => {
+      beforeEach(() => {
+        setupStubs(userExternalId, gatewayAccountId, gatewayAccountExternalId, serviceName)
+        cy.visit(`/account/${gatewayAccountExternalId}/payment-types`)
+      })
+
+      it('should show debit cards that do not require 3DS as enabled and without hint', () => {
+        cy.get('#debit-2').should('be.checked')
+        cy.get('#debit-2-item-hint').should('not.exist')
+      })
+
+      it('should show debit cards requiring 3DS as disabled and with hint', () => {
+        cy.get('#debit-3').should('be.disabled')
```

Comment:
`#debit-3` might be a bit brittle but I cannot immediately think of a better way of doing this